index.js

```diff
@@ -422,8 +445,6 @@ Polyglot.prototype.has = function (key) {
 };
 
 // export transformPhrase
-Polyglot.transformPhrase = function transform(phrase, substitutions, locale) {
-  return transformPhrase(phrase, substitutions, locale);
-};
+Polyglot.transformPhrase = transformPhrase;
```

Comment:
This one is optional. I'm not sure if the api intentionally excludes custom tokens or plural rules here.

Reply:
It does intentionally exclude them.